src/router.js

```diff
@@ -69,19 +69,11 @@ const initialLocale = () => {
 };
 
 // Implements the restoreSession meta field.
-const restoreSession = async (to) => {
-  if (!last(to.matched).meta.restoreSession) return;
-  const sessionExpires = localStore.getItem('sessionExpires');
-  // We send a request if sessionExpires == null, partly in case there was a
-  // logout error.
-  if (sessionExpires != null && parseInt(sessionExpires, 10) <= Date.now())
-    return;
-  await store.dispatch('get', [{
-    key: 'session',
-    url: '/v1/sessions/restore',
-    alert: false
-  }]);
-  await logIn(router, store, false);
+const restoreSessionIfShould = async (to) => {
```

Comment:
I don't love the name of this function, but I couldn't think of anything better. 🤔

Reply:
`tryRestoreSession`? `restoreSessionIdAble`? I guess the "should" comes from it being in the router file, and the router saying "hey we are at the restoreSession route, we _should_ restore". So it's not really trying/checking if it should or should not do something based on some special condition, but it needs to do the whole restore session process because that's what the router says. `reallyRestoreSession` `doRestoreSession` `restoreSessionNow` `routerRestoreSession` `handleRestoreSession`... any of those inspire you?

Reply:
Haha thank you, `routerRestoreSession` got me thinking of a better name! The "should" from before was actually there because some routes won't attempt to restore the session. (It's a configurable route meta field.) With that in mind, I've renamed the function to `restoreSessionForRoute()`. I like that that captures that the function is used specifically in the router and takes the initial route as an argument. Maybe there's a better name out there, but I think I'm satisfied with this new one!